src/pages/OnboardingPage/screens/Introduction.js

```diff
@@ -33,19 +35,38 @@ export default class Introduction extends Component {
 
 	createApp = () => {
 		const { value } = this.input;
-		const pattern = /^[a-zA-Z0-9_+-@$\.]+$/;
+		const isValidAppName = validateAppName(value);
 		let app = {};
 
 		this.setState({
 			status: 'Creating your app... Please wait!',
 			error: '',
 		});
 
+		const validationsList = [
```

Comment:
We should put `validationList` in the helper utils as well.

Reply:
Sounds good 👍 . Updating